1_fetch/src/data_summary.R

```diff
@@ -64,6 +74,43 @@ regress_site <- function(sites)
   return(dfstats)
 }
 
+flexible_linear_regression <- function(sites, type)
```

Comment:
newly modified function that takes two parameters to determine which type of regression is should perform

Reply:
my review: this function will need to be changed again to edit the values in the output data frame, most notably next i will need to change maxes, means and monthly values to reflect those of the means that are used to group the data, and not the daily values, since daily values aren't being used as the residuals.